.github/workflows/release.yml

```
@@ -55,6 +55,9 @@ jobs:
 
       - name: Update version
         run: |
+          sudo touch config.yml
+          sudo find . -path "./.git" -prune -o -exec chown 1000:1000 {} \;
+          sudo chown 1000:1000 .
```

@mmunro-ltrr does this need to be recursive
```suggestion
 sudo chown --recursive 1000:1000 .
 ```

---

Either way I am fine with merging

---

The same step in the review site workflow isn't recursive.